tests/snuba/api/endpoints/test_organization_events_stats_mep.py

```diff
@@ -1150,3 +1153,36 @@ def test_top_events_with_transaction_on_demand_and_no_environment(self):
             group, agg, row1, row2 = group_count
             row_data = response.data[group][agg]["data"][:2]
             assert [attrs for time, attrs in row_data] == [[{"count": row1}], [{"count": row2}]]
+
+    def _test_is_metrics_extracted_data(
+        self, params: dict[str, Any], expected_on_demand_query: bool, dataset: str
+    ) -> None:
+        features = {"organizations:on-demand-metrics-extraction": True}
+        spec = OnDemandMetricSpec(
+            field="count()",
+            query="transaction.duration:>1s",
+            spec_type=MetricSpecType.DYNAMIC_QUERY,
+        )
+
+        self.store_on_demand_metric(1, spec=spec)
+        response = self.do_request(params, features=features)
+
+        assert response.status_code == 200, response.content
+        meta = response.data["meta"]
+        # This is the main thing we want to test for
+        assert meta.get("isMetricsExtractedData", False) is expected_on_demand_query
+        assert meta["dataset"] == dataset
+
+        return meta
+
+    def test_is_metrics_extracted_data_is_included(self):
+        self._test_is_metrics_extracted_data(
+            {
+                "dataset": "metricsEnhanced",
+                "query": "transaction.duration:>=91",
+                "useOnDemandMetrics": "true",
+                "yAxis": "count()",
+            },
```

Comment:
Almost identical params to the `events` endpoint except this one does not require `field`. See test:
https://github.com/getsentry/sentry/blob/8eba43d1c1d3288382b7de1d1c63273ce75410f2/tests/snuba/api/endpoints/test_organization_events_mep.py#L2634-L2640